tendrl/node_agent/message/handler.py

```diff
@@ -0,0 +1,44 @@
+from _socket import error as _socket_error
+import gevent.event
+import gevent.greenlet
+from gevent.server import StreamServer
+from io import BlockingIOError
+import json
+from tendrl.commons.message import Message
+from tendrl.node_agent.message.logger import Logger
+
+
+RECEIVE_DATA_SIZE = 4096
+
+
+class MessageHandler(gevent.greenlet.Greenlet):
```

Comment:
Needs to be updated based on https://github.com/Tendrl/commons/pull/134#discussion_r97725477